packages/yt-dlp-ejs/build.sh

```diff
@@ -0,0 +1,26 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/yt-dlp/ejs
+TERMUX_PKG_DESCRIPTION="External JavaScript for yt-dlp supporting many runtimes"
+TERMUX_PKG_LICENSE="Unlicense"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.3.1"
+TERMUX_PKG_SRCURL="https://github.com/yt-dlp/ejs/releases/download/$TERMUX_PKG_VERSION/yt_dlp_ejs-$TERMUX_PKG_VERSION.tar.gz"
+TERMUX_PKG_SHA256=7f2119eb02864800f651fa33825ddfe13d152a1f730fa103d9864f091df24227
+TERMUX_PKG_PYTHON_COMMON_DEPS="build, hatchling, hatch-vcs"
+TERMUX_PKG_PLATFORM_INDEPENDENT=true
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS='nodejs | nodejs-lts, python'
+if (( TERMUX_ARCH_BITS == 64 )); then
+	TERMUX_PKG_DEPENDS='deno | nodejs | nodejs-lts, python'
+fi
```

Comment:
This only occurred to me after hitting the merge button.

Is this even gonna work for 32 bit architectures if the package is architecture independent,
or are we just satisfying the constraints of the buildorder workflow?

The `yt-dlp-ejs_0.3.1_all.deb` package has the following in its control file.
```yml
Package: yt-dlp-ejs
Architecture: all
Installed-Size: 296
Maintainer: @termux
Version: 0.3.1
Homepage: https://github.com/yt-dlp/ejs
Depends: deno | nodejs | nodejs-lts, python
Description: External JavaScript for yt-dlp supporting many runtimes
```

I don't have any 32 bit device to verify this...